{"repository":"commons-math","prod_path":"src/main/java/org/apache/commons/math4/exception/util/ExceptionContext.java","test_path":"src/test/java/org/apache/commons/math4/exception/util/ExceptionContextTest.java","prod_time":"2015-08-29 05:42:39","test_time":"2020-10-01 04:48:42","add_annotation_line":0,"add_call_line":4,"add_classname_line":0,"add_condition_line":0,"add_field_line":0,"add_import_line":0,"add_packageid_line":0,"add_parameter_line":0,"add_return_line":0,"del_annotation_line":0,"del_call_line":4,"del_classname_line":0,"del_condition_line":0,"del_field_line":0,"del_import_line":0,"del_packageid_line":0,"del_parameter_line":0,"del_return_line":0,"label":"NEGATIVE","pro_Commit":"392323e2d44bacdac605c1a4f037f06e21dcc239","test_Commit":"2d14b517c3beaeaa6766ede9f14cffa416cfeb52","true_label":"NEGATIVE"}